starknet_py/net/account/account.py

```diff
@@ -246,28 +250,36 @@ async def _prepare_invoke_v3(
         )
         return _add_resource_bounds_to_transaction(transaction, resource_bounds)
 
-    async def _estimate_fee(
+    async def estimate_fee(
         self,
-        tx: AccountTransaction,
+        tx: Union[AccountTransaction, List[AccountTransaction]],
+        skip_validate: bool = False,
         block_hash: Optional[Union[Hash, Tag]] = None,
         block_number: Optional[Union[int, Tag]] = None,
-    ) -> EstimatedFee:
+    ) -> Union[EstimatedFee, List[EstimatedFee]]:
         """
-        :param tx: Transaction which fee we want to calculate.
+        Estimate fee for transaction
+
+        :param tx: Transactions which fee we want to calculate.
+        :param skip_validate: Flag checking whether the validation part of the transaction should be executed.
         :param block_hash: a block hash.
         :param block_number: a block number.
-        :return: Estimated fee.
+        :return: Estimated fee of transactions.
         """
```

Comment:
The docstring here seems redundant now, as having it in the `BaseClass` already provides the necessary information.